dask_expr/collection.py

```diff
@@ -671,6 +671,12 @@ def rename(self, columns):
         return new_collection(expr.RenameFrame(self.expr, columns=columns))
 
     def explode(self, column):
+        if (
+            column is not None
+            and not isinstance(column, list)
+            and not hasattr(column, "dtype")
+        ):
+            column = [column]
```

Comment:
**Just an observation**: This pretty much matches the `subset` normalization used for #173. Will this logic be repeated in more places? If so, maybe we should add/use a simple utility to always convert to `list`? Could information ever be lost by converting `pd.Index` -> `list`?

Reply:
Yes this is the same, adding a utility is on my todo list, can also do it in this PR if you prefer.

Yeah, but nothing that's relevant here fortunately.

Reply:
Updated to create a utility function